frontend/src/App.vue

```diff
@@ -50,5 +64,28 @@ h1, h2, h3, h4, h5, h6
       <router-view/>
     </keep-alive>
     <app-footer/>
+    <div id="consent" v-if="askConsent">
+      We log and analyze site traffic. Read our <router-link to="/privacy">privacy policy</router-link>.
```

Comment:
`log and analyze` kuulostaa aika yksityiskohtaiselta. Miten olisi vain `track` tai `monitor`?